Yatima/Transpiler/TranspileM.lean

```diff
@@ -10,17 +9,17 @@ structure State where
   visited : Std.RBTree Name compare
   deriving Inhabited
 
+open Yatima.Compiler Yatima.FromIpld
 
-def State.getStringBindings (s : State) : List (Name × Lurk.Expr) :=
-  s.appendedBindings.data
-
-open Yatima.Compiler
-open Yatima.FromIpld
-
-abbrev TranspileM := ReaderT CompileState $ EStateM String State
+abbrev TranspileM := ReaderT CompileState $
+  ExceptT TranspileError $ StateT State IO
 
 def appendBinding (b : Name × Lurk.Expr) : TranspileM Unit := do
-  dbg_trace s!"append {b.1}"
+  IO.println "\n========================================="
```

Comment:
For future us: we should add a `--verbose` flag to the CLI that controls whether this IO prints or not.